src/pages/queue.js

```diff
@@ -65,7 +75,23 @@ class Queue extends React.Component {
     const locationText = this.props.queue.location || 'No location specified'
     return (
       <Container fluid>
-        <h3>{this.props.queue.name}</h3>
+        <h3>
+          {this.props.queue.isConfidential && (
+            <span>
+              <FontAwesomeIcon
+                icon={faEyeSlash}
+                fixedWidth
+                className="mr-2"
+                id="confidentialIcon"
+              />
+              <UncontrolledTooltip placement="bottom" target="confidentialIcon">
+                This is a confidential queue! Students won&apos;t be able to see
```

Comment:
It'd be nice if we could customize this message for students - rephrase it to `You won't be able to see questions from other students`.

Reply:
What do you think about just rephrasing the entire message?  So make it `This is a confidential queue! {Students/You} won't be able to see questions from other students.`

Reply:
I like it!